src/events/balancer.py

```diff
@@ -23,7 +23,7 @@
     USER,
     Status,
 )
-from managers.balancer import BalancerManager
+from managers.balancer import BalancerManager, RebalanceMode
```

Comment:
Shall we make `RebalanceMode` a proper data structure in `literals.py`?

Never mind, as soon as I wrote it, I strongly thought so.